regolith/helpers/a_proposalhelper.py

```diff
@@ -34,45 +34,51 @@ def subparser(subpi):
     subpi.add_argument("-a", "--authors",
                        help="Other investigator names"
                        )
-    subpi.add_argument("--begin_date", nargs="+",
-                       help="The start date for the proposed grant in format YYYY-MM-DD."
+    subpi.add_argument("--begin_day", nargs="+",
+                       help="The start day for the proposed grant in format YYYY-MM-DD."
+                       )
+    subpi.add_argument("--begin_month", nargs="+",
+                       help="The start month for the proposed grant in format YYYY-MM-DD."
+                       )
+    subpi.add_argument("--begin_year", nargs="+",
+                       help="The start year for the proposed grant in format YYYY-MM-DD."
                        )
     subpi.add_argument("--end_date", nargs="+",
                        help="The end date for the proposed grant in format YYYY-MM-DD."
                        )
-    subpi.add_argument("--call", "--call_for_proposals", nargs="+",
+    subpi.add_argument("--call_for_proposals", nargs="+",
                        help="Url for call for proposals"
                        )
-    subpi.add_argument("-cpp", "--cpp_info", nargs="+",
+    subpi.add_argument("--cpp_info", nargs="+",
```

Comment:
this will be too hard for me because I don't know the keys off the top of my head.